Topic metadata feature python sdk (#13381)
ydb-platform/ydb: ydb/docs/en/core/reference/ydb-sdk/topic.md

  List<MetadataItem> metadata = message.getMetadataItems();
  ```

- Python

>   To take advantage of message metadata feature, create the `TopicWriterMessage` object with `metadata_items` argument as below:

Reviewer: ```suggestion
  To write a message that includes metadata, create the `TopicWriterMessage` object with the `metadata_items` argument:
```

Author: done

  ```python

>   mess = ydb.TopicWriterMessage(data=f"mess-{i}", metadata_items={"meta-key": "meta-value"})

Author: done

  writer.write(mess)
  ```

>   While reading, metadata can be received from a `PublicMessage` with the `metadata_items` field:

Reviewer: ```suggestion
  While reading, get metadata from the `metadata_items` field of the `PublicMessage` object:
```

Author: done
